### PR TITLE
Tests config labels

### PR DESCRIPTION
**Pull request by matejvasek**
# Changes

:test_tube: added test for label config

this adds tests for interactive wizard for label addition and deletion using `func config labels [add|remove]`

### CODE REVIEW

**Comment by jrangelramos** [comment]
Hi @matejvasek one thing I notice when running it from terminal (in verbose mode) is that it seems it is drops the control that are captured by the terminal after tests are completed.
```
 $ go test ./cmd -v
...
PASS
ok  	knative.dev/kn-plugin-func/cmd	1.503s
$ ;238R;34R;238R;35R;238R;34R;238R;35R;238R;34R;238R;35R
```
is that expected? Tests pass anyway

**Comment by matejvasek** [comment]
@jrangelramos I've just removed output echoing. It should fix it.

File: cmd/config_labels_test.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+func (m *mockFunctionLoaderSaver) Load(path string) (fn.Function, error) {
+	return m.f, nil
+}
+
+func (m *mockFunctionLoaderSaver) Save(f fn.Function) error {
+	m.f = f
+	return nil
+}
+
+func assertLabelEq(t *testing.T, actual fn.Pairs, want fn.Pairs) {
```
Comment (lkingland) [inline cmd/config_labels_test.go:30]: Perhas a [`reflect.DeepEqual` ](https://pkg.go.dev/reflect#DeepEqual)Here would make this shorter, unless order is not guaranteed...

File: cmd/config_labels_test.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	p := func (k,v string) fn.Pair {
+		return fn.Pair{Name: &k, Value: &v}
+	}
+
+	assertLabel := func (ps fn.Pairs) {
+		t.Helper()
+		assertLabelEq(t, *labels, ps)
+	}
+
+	run("add", enter, "a", enter, "b", enter)
```
Comment (lkingland) [inline cmd/config_labels_test.go:129]: Being able to test the result of interactive commands is quite a nice addition!

**Review by lance — APPROVED** [review]
Love it -thanks!
/lgtm

**Comment by matejvasek** [comment]
@lkingland @lance I rebased PTAL

**Review by lance — COMMENTED** [review]
Great addition -thanks!

**Pull request merged**